src/main/groovy/de/dkfz/eilslabs/tools/execution/ExecutionHelper.groovy

```diff
@@ -0,0 +1,97 @@
+
+/*
+ * Copyright (c) 2016 eilslabs.
+ *
+ * Distributed under the MIT License (license terms are at https://www.github.com/eilslabs/Roddy/LICENSE.txt).
+ */
+
+package de.dkfz.eilslabs.tools.execution
+
+import de.dkfz.eilslabs.tools.logging.LoggerWrapper
+
+import java.lang.reflect.Field
+
+@groovy.transform.CompileStatic
+class ExecutionHelper {
+    private static final LoggerWrapper logger = LoggerWrapper.getLogger(ExecutionHelper.class.name);
+
+    public static String getProcessID(Process process) {
+        Field f = process.getClass().getDeclaredField("pid");
+        f.setAccessible(true);
+        String processID = f.get(process)
+        return processID
+    }
+
+    static class ExtendedProcessExecutionResult {
+        int exitValue;
+        String processID;
+        List<String> lines = [];
+
+        ExtendedProcessExecutionResult(int exitValue, String processID, List<String> lines) {
+            this.exitValue = exitValue
+            this.processID = processID
+            this.lines = lines
+        }
+    }
+
+    public static String executeSingleCommand(String command) {
+        //TODO What an windows systems?
+        //Process process = Roddy.getLocalCommandSet().getShellExecuteCommand(command).execute();
+        Process process = command.execute();
+
+        final String separator = System.getProperty("line.separator");
+        process.waitFor();
+        if(process.exitValue()) {
+            throw new RuntimeException("Process could not be run" + separator + "\tCommand: sh -c "+ command + separator + "\treturn code is: " + process.exitValue())
+        }
+
+        def text = process.text
+        return text.length() > 0 ? text[0 .. -2] : text; //Cut off trailing "\n"
```

Comment:
Should that not be 
```
public static chomp(String text) {
  text.length() >= 2 ? text[0..-2] : text
}
```

I.e. a different condition (and maybe a dedicated function).